pkg/resource/endpoint/testdata/test_suite.yaml

```diff
@@ -0,0 +1,17 @@
+tests:
+  - name: "Endpoint demo test"
+    description: "Endpoint CRD tests"
+    scenarios:
+     - name: "Create=InvalidInput"
+       description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
+       given:
+         desired_state: "endpoint/cr/e_cmd_invalid_before_create.yaml"
+         svc_api:
+           - operation: CreateEndpointWithContext
+             error:
+               code: InvalidParameterValue
+               message: "The endpoint name must start with an alphanumeric character."
+       invoke: Create
```

Comment:
Have you tried testing delete/update for any resource ? What is the plan for those tests ?

Reply:
The delete and readone tests are further down. I have not attempted an update test, though the format should be similar to a create or readone test.

Reply:
Saw the delete test later yes, thanks!